website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -80,6 +80,7 @@ By supplying the necessary attributes under a `vcs-repository` object, you can c
 | `data.attributes.speculative-enabled`         | boolean | true      | Whether this workspace allows automatic [speculative plans][]. Setting this to `false` prevents Terraform Cloud from running plans on pull requests, which can improve security if the VCS repository is public or includes untrusted contributors. It doesn't prevent manual speculative plans via the CLI or the runs API.                                                                                                                                                                                    |
 | `data.attributes.terraform-version`           | string  | (special) | The version of Terraform to use for this workspace. This can be either an exact version or a [version constraint](/language/expressions/version-constraints) (like `~> 1.0.0`); if you specify a constraint, the workspace will always use the newest release that meets that constraint. If omitted when creating a workspace, this defaults to the latest released version.                                                                                                                                   |
 | `data.attributes.trigger-prefixes`            | array   | `[]`      | List of repository-root-relative paths which should be tracked for changes, in addition to the working directory.                                                                                                                                                                                                                                                                                                                                                                                               |
+| `data.attributes.trigger-patterns`            | array   | `[]`      | List of repository-root-relative glob patterns which describe files tracked for changes.                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
What about something like this?

```suggestion
| `data.attributes.trigger-patterns`            | array   | `[]`      | List of glob patterns that describe the files Terraform Cloud monitors for changes. Trigger patterns are always appended to the root directory within the repository.                                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Reply:
What about saying `of the repository` instead of `withing the repository`?

Also should we then change wording for prefixes as well to match the same style:
```
List of repository-root-relative paths which should be tracked for changes, in addition to the working directory.
```

Reply:
I went ahead and made changes, let me know if something should be changed.
4227735d18cce384e666ca1acecfc532ba90ae3c